src/analysis/retail/warrior/protection/modules/talents/AngerManagement.tsx

```diff
@@ -26,6 +26,7 @@ class AngerManagement extends Analyzer {
   wastedReduction: { [spellId: number]: number } = {};
   effectiveReduction: { [spellId: number]: number } = {};
   protected spellUsable!: SpellUsable;
+  COOLDOWNS_AFFECTED_BY_ANGER_MANAGEMENT: any[] = [];
```

Comment:
If this is only filled by doing `this.COOLDOWNS_AFFECTED_BY_ANGER_MANAGEMENT.push(e.id)` from slightly further down, this should be typed as `number[]` instead of `any[]`.